app/server/lib/OIDCConfig.ts

```diff
@@ -89,6 +91,10 @@ export class OIDCConfig {
       envVar: 'GRIST_OIDC_IDP_CLIENT_SECRET',
       censor: true,
     });
+    const httpTimeout = section.flag('httpTimeout').readInt({
+      envVar: 'GRIST_OIDC_HTTP_TIMEOUT',
+      defaultValue: 3500,
```

Comment:
You may add a comment telling that this value is also the default value set by the `openid-client` library:
https://github.com/panva/node-openid-client/blob/main/docs/README.md#customizing-http-requests